client/src/App.js

```diff
@@ -18,7 +18,7 @@ function App() {
   return (
     <div className="App">
 
-      <header className="App-header">
+      {/* <header className="App-header"> */}
```

Comment:
@rohinij15 There was a compilation error because there were two `header` elements, and the first one didn't have a closing tag. I will create a bug ticket to get these removed, but commenting them out for now so the app runs